functional-tests/fn_bridge_deposit_reclaim_drt_seen.py

```diff
@@ -0,0 +1,292 @@
+        # since they will be able to spend the DRT output.
+        # We need to wait for the reclaim path 1008 blocks to mature
+        # so that we can use the take back path to spend the DRT output.
+        # Breaking by chunks
+        chunks = 8
+        blocks_to_generate = DEFAULT_TAKEBACK_TIMEOUT // chunks
+        self.logger.debug(f"Generating {DEFAULT_TAKEBACK_TIMEOUT} blocks in {chunks} chunks")
+        for i in range(chunks):
+            self.logger.debug(f"Generating {blocks_to_generate} blocks in chunk {i+1}/{chunks}")
+            btcrpc.proxy.generatetoaddress(blocks_to_generate, UNSPENDABLE_ADDRESS)
+
+        # wait up a little bit
+        time.sleep(1)
+
+        # Make sure that the BTC refund address has the expected balance
+        refund_btc_balance = get_balance(
+            refund_addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Refund BTC balance (before takeback): {refund_btc_balance}")
+        assert refund_btc_balance == 0, "BTC balance is not zero"
+
+        # Check BTC balance of the recovery address
+        btc_recovery_balance_pre = get_balance_recovery(
+            recovery_addr,
+            bridge_pk,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"DRT BTC balance (before takeback): {btc_recovery_balance_pre}")
+
+        # Spend the take back path
+        take_back_tx = bytes(
+            take_back_transaction(refund_addr, bridge_pk, btc_url, btc_user, btc_password)
+        ).hex()
+        self.logger.debug("Take back tx generated")
+
+        # Send the transaction to the Bitcoin network
+        txid = btcrpc.proxy.sendrawtransaction(take_back_tx)
+        self.logger.debug(f"sent take back tx with txid = {txid} for address {el_address}")
+        btcrpc.proxy.generatetoaddress(2, UNSPENDABLE_ADDRESS)
+        time.sleep(1)
+
+        # Make sure that the BTC recovery address has 0 BTC
+        btc_recovery_balance_post = get_balance_recovery(
+            recovery_addr,
+            bridge_pk,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"DRT BTC balance (after takeback): {btc_recovery_balance_post}")
+        assert btc_recovery_balance_post == 0, "BTC balance is not zero"
+
+        # Make sure that the BTC refund address has the expected balance
+        refund_btc_balance_post = get_balance(
+            refund_addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Refund BTC balance (after takeback): {refund_btc_balance_post}")
+        expected_balance_lower_bound = btc_recovery_balance_pre - 100_000_000
+        self.logger.debug(f"Expected balance lower bound: {expected_balance_lower_bound}")
+        assert (
+            refund_btc_balance_post >= expected_balance_lower_bound
+        ), "BTC balance is not as expected"
+
```

Comment:
Shouldn't the expected balance be `DRT amount (1.01 BTC) - transaction fees`? Specifically, what makes the refund address to have a balance greater than this amount?

Reply:
Done in 2b2cc8e6db5166e7342d03ae12c41dfa8917c1ac